src/sortable-list/components/sortable-list.component.ts

```diff
@@ -0,0 +1,167 @@
+import { Component, Input, Output, EventEmitter, forwardRef, animate, style, state, transition, keyframes, trigger } from '@angular/core';
+import { NG_VALUE_ACCESSOR, ControlValueAccessor } from '@angular/forms';
+import 'rxjs/add/operator/first';
+
+import { DraggableItem } from '../models';
+import { DraggableItemService } from '../services';
+
+const nullCallback = (arg?: any): void => { return void 0; };
+
+/* tslint:disable */
+@Component({
+    selector: 'ng2-sortable-list',
+    templateUrl: './sortable-list.component.html',
```

Comment:
Interesting question, I will check it today. In general don't see any problems with html inlining since it is not so big.